content/en/network_performance_monitoring/devices/guide/build-ndm-profile.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Build a NDM Profile
+kind: guide
+further_reading:
+- link: "https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/"
+  tag: "Documentation"
+  text: "Profile format reference"
+- link: "https://datadoghq.dev/integrations-core/tutorials/snmp/sim-format/"
+  tag: "Documentation"
+  text: "Simulation data format reference"
+---
+
+Datadog Network Device Monitoring uses profiles for certain makes and models of network devices. This tutorial shows the steps for building a basic NDM profile that collects OID metrics from HP iLO4 devices.
```

Comment:
nit, but rephrase to: 

Datadog Network Device Monitoring uses profiles for collecting metrics from network devices. These are defined narrowly by a MIB, or if you'd like to collect metrics from specific device make and model. This tutorial ...